Storage.js

```diff
@@ -0,0 +1,26 @@
+/* Referencing the name and email input values */
+const inputs = document.querySelectorAll('input');
+const imput = document.querySelector('input');
+/* if the localStorage object is not empty then pre-fill the inputs with the 'form' string */
+if (localStorage.length !== 0) {
```

Comment:
- [ ] I think this condition in your if statement is a bit elaborate, it checks if the users' whole local storage has an item, this seems to be an issue because I noticed that if I have other objects in my local storage, the preserve data tends not to work properly, I suggest you check if the local storage has an item with the key of `form` which is the name you gave. :+1: :100: :+1: